src/constants/translations/en/common.json

```diff
@@ -38,8 +38,9 @@
   "megabytes": "MB",
   "uploadNewFile": "Upload new file",
   "typeError": "Wrong file type. Only .pdf/.jpg/.jpeg files are allowed.",
-  "allFilesSizeError": "All files size error",
+  "allFilesSizeError": "Size of all files cannot be more than 20 MB",
   "fileSizeError": "Size for one file cannot be more than 5 MB.",
+  "fileNameError":"File name cannot be longer than 55 symbol",
```

Comment:
Maybe "File name cannot be longer than 55 characters" with be better description of error?

Reply:
makes sense, even though user won't ever count his file name length😄
updated✅